pkg/types/defaults.go

```diff
@@ -10,9 +10,10 @@ const (
 
 var (
 	NcpSections      = []string{"DEFAULT", "ha", "k8s", "coe", "nsx_v3", "vc"}
-	AgentSections    = []string{"DEFAULT", "k8s", "coe", "nsx_node_agent", "nsx_kube_proxy"}
+	AgentSections    = []string{"DEFAULT", "nsx_node_agent", "nsx_kube_proxy"}
 	OperatorSections = []string{"DEFAULT", "ha", "k8s", "coe", "nsx_v3", "vc", "nsx_node_agent", "nsx_kube_proxy"}
 	BootstrapOptions = []string{"DEFAULT", "nsx_node_agent"}
+	AgentRestartKeys = []string{"apiserver_host_ip", "apiserver_host_port", "client_token_file", "ca_file", "enable_multus", "enable_hostport_snat", "connect_retry_timeout"}
```

Comment:
it's better to change this as a map as we did for WCPOptions. The good point of map is to better know the map relationship between section and option.

Reply:
> it's better to change this as a map as we did for WCPOptions. The good point of map is to better know the map relationship between section and option.

Thanks, let me check.

Reply:
Agree that map will make it more clear, so update options with structure map, thanks.